jade-interaction.el

```diff
@@ -142,7 +158,9 @@ open, and set it in the current buffer."
   (let ((map (make-sparse-keymap)))
     (define-key map (kbd "C-x C-e") #'jade-eval-last-node)
     (define-key map (kbd "C-c M-i") #'jade-inspect-last-node)
+    (define-key map (kbd "C-c C-z") #'jade-switch-to-repl-buffer)
```

Comment:
Is this the same keybinding as in CIDER?

Also, I would like to have in all Jade buffers (inspectors, debugger, etc.)